ideascube/templates/ideascube/includes/cards/deus-ex-silicium.html

```diff
@@ -1,3 +1,2 @@
 {% load i18n %}
-{% trans "learn" as theme %}
-{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme_class="learn" name="Deus Ex Silicium" description="Vidéos dédiées à la science des semi-conducteurs" theme=theme %}
+{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme="learn" %}
```

Comment:
It would probably have been better to split this `theme_class` vs `theme` change in its own commit, since it's not related to the package+card management change.